pkgs/applications/networking/eddie-ui/default.nix

```diff
@@ -0,0 +1,92 @@
+{ stdenv,
+  fetchFromGitHub,
+  clang,
+  cmake,
+  mono,
+  dotnet-sdk,
+  which,
+  glibc,
+  pkgconfig,
+  gtk2,
+  libappindicator-gtk2,
+  lzma,
+  libselinux,
+  libsepol,
+  fribidi,
+  libthai,
+  libdatrie,
+  xorg,
+  openvpn,
+  stunnel
+  }:
+
+stdenv.mkDerivation rec {
+  pname = "eddie-ui";
+  version = "2.18.9";
+  arch = if stdenv.system == "i686-linux"
+    then "x86"
+    else "x64";
+  src = fetchFromGitHub {
+    owner = "AirVPN";
+    repo = "Eddie";
+    rev = "2.18.9";
+    sha256 = "1nvnvy6lskydfrlsrbyk065hl6zxdyivhrynydny1kirz3qmrw61";
+  };
+  sourceRoot= "/build/source/src/UI.GTK.Linux.Tray";
+  buildPhase = ''
+    chmod -R +w /build/source
+    cd /build/source/src/
+    find ./ -type f -iname "*.sh" -exec chmod +x {} \;
+    patchShebangs ./
+
+    # Compile C# sources
+    cd /build/source
+    xbuild /verbosity:minimal /p:Configuration="Release" /p:Platform="$arch" src/eddie2.linux.sln
```

Comment:
This is only available on Darwin.

Reply:
Sorry, didn't get it. Could you please explain? Build is fine on NixOS on my Dell laptop.